pull_request_template.md

```diff
@@ -6,13 +6,13 @@ ___Add your PR description here, deleting this example___
 
 [Loom Video](Add your loom video link here)
 
-#### Trello Link
+#### Jira Link
 
-___Paste your trello card link here, deleting this example___
+___Paste your jira ticket link here, deleting this example___
 
 (Example below):
 
-<blockquote class="trello-card"><a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;JfipwNkb&#x2F;265-fix-refactor-sidebar-to-have-icons-representing-each-element-in-the-sidebar">Fix: Refactor sidebar to have icons representing each element in the sidebar</a></blockquote>
+https://bloomtechlabs.atlassian.net/browse/BL-637
```

Comment:
Are we supposed to be updating the template every time we submit a pull request? My assumption has been that we leave the template and fill out the form on github for each pull request.